decks/react/02-props.mdx

```diff
@@ -0,0 +1,79 @@
+import { FullScreenCode } from "mdx-deck/layouts";
+
+## Props
+
+- Data input from other components within the application
```

Comment:
- Props define the 'API' of the component and how it interacts within your application
- Props become the attributes on your component that you use to pass in data 
- Components are just functions, that receive the `props` object